.github/workflows/sanity.yml

```
@@ -0,0 +1,20 @@
+name: "Codeowners Validator"
+
+on:
+  schedule:
+    # Runs at 08:00 UTC every day
+    - cron:  '0 8 * * *'
```

```suggestion
    - cron:  '0 8 * * *'
  workflow_dispatch:
```
I came to love the `workflow_dispatch` trigger, especially for scheduled workflows, since it allows you to trigger the workflow whenever you want.
That said it might also be good to run it on `push` and/or `pull_request` in case the `CODEOWNERS` file gets changed e.g. if we add a new subpackage with specific owners.
Since the workflow isn't coupled to other steps and should be pretty quick this shouldn't produce a big overhead.